packages/sdk/tests/AbstractConfigManager.test.ts

```diff
@@ -424,6 +424,36 @@ describe("AbstractConfigManager", async () => {
             });
         });
     });
+    describe("promptForDeployDirectory", () => {
```

Comment:
(nitpick) seems like there are some redundant new lines here, not requesting changes but just something I noticed 😅